src/Diagnostics.DataProviders/Interfaces/IAscClient.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+
+namespace Diagnostics.DataProviders.Interfaces
+{
+    public interface IAscClient
+    {
+        /// <summary>
+        /// Submit a request for ADS insight, keep polling and then retrieve the Insight result once insight processing is complete.
+        /// </summary>
+        /// <param name="jsonPostBody">POST body in JSON format to submit to Azure Support Center ADS insight.</param>
+        /// <param name="apiVersion">Api version to use which calling Azure Support Center.</param>
+        /// <param name="cancellationToken">Cancellation Token for this operation.</param>
+        /// <returns>Result of the Asc Insight.</returns>
+        //Task<T> SubmitAndGetInsight<T>(string jsonPostBody, string apiVersion, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Has this code been moved, can the commented-out code be deleted?

Reply:
Yeah, this mas been moved to the provider code.. I've removed it. This slipped by me.